using-valtimo/plugin/documenten-api/configure-plugin.md

```diff
@@ -0,0 +1,53 @@
+# Using the Documenten API Plugin
+
+The Documenten API plugin can be used to store documents and their metadata in an application that supports the
+Documenten API standard created and used by the Dutch government.
+
+## Configure the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Documenten API plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the Documenten API dependency can be found
+[here](../../../valtimo-implementation/modules/documenten-api.md)
+
+To configure this plugin the following properties have to be entered:
+- The `Documenten API URL` property should contain the complete base URL of the Documenten API to connect to. This 
```

Comment:
Not sure how I feel about regular fields being displayed as code block. Maybe try a structure like
- **Document API URL**. Conttains the complete base URL of the Documenten API to connect to.

(And repeat for the rest)